content/lessons/chapter-2/scripting-2.tsx

```diff
@@ -121,11 +125,18 @@ const config: EditorConfig = {
 export default function Scripting2({ lang }) {
   const t = useTranslations(lang)
 
+  const [language, setLanguage] = useState(config.defaultLanguage)
+
+  const handleLanguageCopy = (language) => {
+    setLanguage(language)
+  }
+
   return (
     <ScriptingChallenge
       lang={lang}
       config={config}
       successMessage={t('chapter_two.scripting_two.success')}
+      languageCopy={handleLanguageCopy}
```

Comment:
Let's rename this to `onCopyLanguage` so you know from the prop's name that it should be a function, this is the way its usually done in React